food-truck-api/src/main/java/food/truck/api/endpoint/TruckEndpoint.java

```
@@ -148,13 +169,20 @@ public void deleteTruck(@AuthenticationPrincipal User u, @PathVariable long truc
                 Optional.ofNullable(data.schedule),
                 Optional.ofNullable(data.foodCategory)
         ));
+        try {
+            indexingService.initiateIndexing();
+        }catch (InterruptedException e){
+            return truck;
```

Swallowed exception again